docs/30_commands.md

```diff
@@ -346,14 +355,16 @@ Delete a Marketplace item
 Delete a single Marketplace item
 
 You need to specify either:
-- the itemId and the version, respectively  (recommended)
+- the itemId and the version, via the respective flags (recommended)
```

Comment:
Do we need the complete tuple here?

Reply:
Yes, but also here is written as a foot note.

Reply:
clarified